joints_limit:
  gripper:
    finger_R_joint:
      has_position_limits: true
      min_position: 0
      max_position: 0.04
      has_velocity_limits: true
      max_velocity: 6.159937255
      has_acceleration_limits: false
      has_jerk_limits: false
      has_effort_limits: true
      max_effort: 0.5293
    finger_L_joint:
      has_position_limits: true
      min_position: 0
      max_position: 0.04
      has_velocity_limits: true
      max_velocity: 6.159937255
      has_acceleration_limits: false
      has_jerk_limits: false
      has_effort_limits: true
      max_effort: 0.5293
    finger_3rd_joint:
      has_position_limits: true
      min_position: 0
      max_position: 0.035
      has_velocity_limits: true
      max_velocity: 6.159937255
      has_acceleration_limits: false
      has_jerk_limits: false
      has_effort_limits: true
      max_effort: 0.5293
